driver/normalizer/normalizer.go

```diff
@@ -316,6 +506,43 @@ var Normalizers = []Mapping{
 			},
 		),
 	)),
+
+	MapSemantic("Parameter", uast.Argument{}, MapObj(
+		Obj{
+			"Identifier": Obj{
+					// Could be identifier or some keywords like __arg, same fields in any case
+					uast.KeyType: Any(),
+					uast.KeyPos:      Var("id_param_pos"),
+					"LeadingTrivia":  Any(),
+					"TrailingTrivia": Any(),
+					"IsMissing": Bool(false),
+					"Text":      Var("name"),
+					"Value":     Var("name"),
+					"ValueText": Var("name"),
+				},
+			"AttributeLists":     Any(),
+			"Default":            Var("def_init"),
+			"IsMissing":          Any(),
```

Comment:
Please assert it to `false`. For Identifier this flag was indicating that it's empty, for example.

Reply:
Done.